UnitsNet/CustomCode/QuantityParser.cs

```diff
@@ -124,7 +124,7 @@ static QuantityParser()
             bool matchEntireString = true)
             where TUnitType : Enum
         {
-            var unitAbbreviations = _unitAbbreviationsCache.GetUnitAbbreviations(unit, formatProvider);
+            var unitAbbreviations = UnitAbbreviationsCache.GetUnitAbbreviations(unit, formatProvider);
```

Comment:
This is one of the reasons I don't favor properties over fields, it's harder to tell from reading the code that we are using a private member vs something public.